libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -147,22 +147,10 @@ - (void)tearDown {
 - (void)testAccountIdentityEquality {
     NSDictionary *accountIdentityMatrix = @{
                                             @"MatchGroup1": @[
```

Comment:
I updated this test case, since SFUserAccountIdentity initialization with `nil` should not be supported... Let me know what do you folks think...